dub.sdl

```diff
@@ -0,0 +1,40 @@
+name "mir-stat"
+description "Dlang Statistical Package"
+
+authors "Ilya Yaroshenko" "John Michael Hall"
+copyright "Copyright © 2016 - 2019, Ilya Yaroshenko; see also information per file."
+license "BSL-1.0"
+
+dependency "mir-core" version=">=1.1.7"
+dependency "mir-algorithm" version=">=3.9.2"
+
+buildType "unittest" {
+	buildOptions "unittests" "debugMode" "debugInfo"
```

Comment:
let use spaces everywhere

Reply:
I had copied this from mir-algorithm so the tabs should be in there as well.